### DISCUSSION TITLE
Extract music symbols from music scores

### DISCUSSION THREAD

**Discussion by kwon-young** [jsvine/pdfplumber#758] [Ask for help with specific PDFs]
Hello,

I am trying to use this library to extract all music symbols from a pdf produced by music typesetting programs like musescore such as this one [1-musescore.pdf](https://github.com/jsvine/pdfplumber/files/10047246/1-musescore.pdf).

However, it seems that the bounding boxes of music symbols are way off as shown in the image produced by pdfplumber.
Especially, the top position and height of each music symbols are way off:

![1-musescore](https://user-images.githubusercontent.com/2647934/202857239-ed4acd63-1122-4436-859f-fa7554484aac.jpg)

Is this expected behaviour or is there a problem in pdfplumber ?

**Comment by jsvine** [comment]
Ah, very interesting! Thanks for flagging. At first glance, this seems likely to be an issue with how [`pdfminer.six`](https://github.com/pdfminer/pdfminer.six) (this repository's main dependency) parses the PDF. Are you experiencing this problem with all music scores, or just some? If there are any with *correct* bounding boxes, attaching those here would also be helpful.

**Comment by kwon-young** [comment]
Hello,
thank you for your reply.

I get the same error with all other music scores and with multiple music typesetting programs (musescore and lilypond).
I got to the same conclusion as you that it is a problem with `pdfminer.six`.
I'll try to open an issue there about this problem.

**Reply by kwon-young** [reply]
Here is the relevant issue on pdfminer.six: https://github.com/pdfminer/pdfminer.six/issues/837
